source/features/review-comments-hidden-indicator.css

```diff
@@ -0,0 +1,41 @@
+tr.rgh-comments-indicator {
+	position: relative;
+}
+tr.rgh-comments-indicator td {
+	position: absolute;
+}
+
+tr.rgh-comments-indicator td::after {
+	background: currentColor;
+	content: '';
+	height: 1px;
+	position: absolute;
+	top: 0;
+	left: 0;
+	width: 200%;
+}
+
+.show-inline-notes tr.rgh-comments-indicator td::after {
+	background: transparent;
+}
+
+tr.rgh-comments-indicator button {
+	position: absolute;
+	left: 0;
+	transform: translate(-100%, -50%);
+	padding: 5px 10px;
+	background: none;
+	color: inherit;
+	border: none;
+	display: flex;
+	align-items: center;
+	line-height: 1;
+}
+
+tr.rgh-comments-indicator button:hover {
+	color: #586069;
+}
+
+.show-inline-notes .rgh-comments-indicator {
+	display: none;
```

Comment:
This will remove the ability to toggle comment through indicators. I don't think we should do that. The original feature request also suggested so:
> It'd also be useful to be able to expand/collapse comments on those placeholders.

Reply:
I missed that part, I was working on the assumption that they only appear when the comments are hidden.

The idea isn't bad, but I think it makes the UI more busy.

Reply:
Perhaps making the underline a bit lighter will improve the UI.

Reply:
Revert cd5abfc8ba1fb74e446b7ebb33826c39ec099fd4?

Reply:
@fregante 	We may want to resolve this first?